### ISSUE TITLE
Fluxion not capturing handshakes 

### ISSUE THREAD

**Issue by Muturuc** [FluxionNetwork/fluxion#476]
----

#### Read the [wiki](https://github.com/FluxionNetwork/fluxion/wiki)?
No*

#### Do you want to request a *feature* or report a *bug*?
*Bug*

#### What is the current behaviour?
Not picking up handshakes 

#### If the current behaviour is a bug, please provide the steps to reproduce it.
I select the network, then wlan0 and I use aircrack option then the windows are open and on my phone it kicks me out of the network but it's not capturing anything 

#### What do you think the expected behaviour should be?
It should work 

#### Specs
*Paste the output of `./script/diagnostics.sh [interface]`*

-----

**Comment by djmeero** [comment]
More details?

**Comment by rad4day** [comment]
I decided to fix your "I commented out everything" bug.
 We still cant help you without more information. If it kicks you out of the network it seems to work fine in this regard. You could use tcpdump to verify if an handshake is picked up by your network card. If it is and fluxion doesn't get it, that would be a bug.
Also generally it's prefered to use pyrit for verifying the handshake.

**Comment by usama7628674** [comment]
@Muturuc You can alse verify hanshakes `pyrit -r XX-XX-XX-XX-XX-XX.cap analyze`.It must say good otherwise recapture it and use pyrit as advised by @rad4day

**Comment by MPX4132** [comment]
It seems this **isn't** an issue related to deauthentication uptime (too aggressive by running for too long). I've verified by checking the timeout and sleep intervals of the deauthenticators.

aireplay-ng timeout & sleep:
https://github.com/FluxionNetwork/fluxion/blob/f1650e0f0764da990d4482075ea1a1c428815b45/attacks/Handshake%20Snooper/attack.sh#L182
mdk3 timeout & sleep:
https://github.com/FluxionNetwork/fluxion/blob/f1650e0f0764da990d4482075ea1a1c428815b45/attacks/Handshake%20Snooper/attack.sh#L188

Other possible causes could be a slow system that can't pick up the handshakes immediately after deauthentication stops. I've verified it works fine, and I've received confirmation over Discord suggesting that might be the case, since running the attack on monitor-only mode, triggering a disconnection manually, and reconnecting after, successfully catches the handshake.
